app/views/nsm/adjustments/show.html.erb

```diff
@@ -9,7 +9,9 @@
         <%= t('.adjusted_costs') %>
       </h2>
 
-      <%= govuk_button_link_to( t('.delete_all_adjustments'), confirm_deletion_nsm_claim_adjustments_path(claim), secondary: true,  data: { turbo: "false" }) %>
+      <% unless claim.assessed? %>
```

Comment:
I think this is the wrong check. In other places in these views we're only showing options to interact if `claim.editable_by?(current_user)`. We have a ticket to replace this rather ad-hoc authorisation with Pundit, but I think for now we do rely on views checking that the current user is allowed to edit a claim before showing editing options. (`editable_by?` checks things like "is the claim assigned to the current user?" as well as whether the claim is assessed or not).